packages/cursorless-vscode/src/storedTargetHighlighter.ts

```diff
@@ -0,0 +1,51 @@
+import { StoredTargetKey, groupBy, toCharacterRange } from "@cursorless/common";
+import { StoredTargetMap } from "@cursorless/cursorless-engine";
+import {
+  ScopeRangeType,
+  ScopeVisualizerColorConfig,
+} from "@cursorless/vscode-common";
+import { VscodeIDE } from "./ide/vscode/VscodeIDE";
+import { vscodeApi } from "./vscodeApi";
+import { VscodeFancyRangeHighlighter } from "./ide/vscode/VSCodeScopeVisualizer/VscodeFancyRangeHighlighter";
+import { getColorsFromConfig } from "./ide/vscode/VSCodeScopeVisualizer/getColorsFromConfig";
+import { mapValues } from "lodash";
+
+const targetColorMap: Partial<Record<StoredTargetKey, ScopeRangeType>> = {
+  instanceReference: "domain",
```

Comment:
A bit of a hack for now, but works well. We can add other colors for these specific targets in the future